examples/pigweed-app/nrfconnect/prj.conf

```diff
@@ -85,3 +85,5 @@ CONFIG_STD_CPP17=y
 # Add support for Zephyr console component to use it for Pigweed console purposes
 CONFIG_CONSOLE_SUBSYS=y
 CONFIG_CONSOLE_GETCHAR=y
+CONFIG_CONSOLE_GETCHAR_BUFSIZE=64
```

Comment:
If small buffer causes an issue, maybe it's worth using a little bit bigger for GETCHAR like 256? PUTCHAR probably isn't as important because the app will probably retry when the buffer is full.

Reply:
I thought about it, but it is hard to predict what size will be correct until we don't know how pigweed is going to use the console (e.g. what is the standard command length). After all it's just sample and I didn't want to increase size without a reason (especially considering current memory optimization works), but if you think it's better approach to make sure that it won't fail for someone, I will change it.

Reply:
You're right, but it wasn't so easy to investigate so I wonder what if someone hits the same issues while trying to send a more sophisticated command. Did you maybe try to send the echo command with payload longer than 64B?

Reply:
I checked that putchar buffer size doesn't really matter, so I left it with default value of 16 B.